src/web/js/trove/chart-lib.js

```diff
@@ -771,6 +772,59 @@
     };
   }
 
+  function intervalChart(globalOptions, rawData) {
+    const table = get(rawData, 'tab');
+    const data = new google.visualization.DataTable();
+    data.addColumn('number', 'x');
+    data.addColumn('number', 'values');
+    data.addColumn({id: 'i0', type: 'number', role: 'interval'});
+    data.addColumn({id: 'i1', type: 'number', role: 'interval'});
+
+    const style = get(rawData, 'style');
+
+    const stickColor = get_default_color(rawData);
+    const stickWidth = get(rawData, 'lineWidth');
+
+    const pointColor = get_pointer_color(rawData);
+    const pointSize = get(rawData, 'point-size');
+
+    const fillOpacity = (style == 'boxes') ? 0 : 1;
+
+    data.addRows(table.map(row => [row[0], row[1], row[1], row[2]]));
```

Comment:
It's likely that the data should not be passed through uninterpreted. Probably it would be correct to call `toFixnum` on all of these if they are all numbers (and matching the columns above). e.g. https://github.com/brownplt/code.pyret.org/blob/08ad7d3e876030e0013f8b889d0d3e9d2d99d4ea/src/web/js/trove/chart-lib.js#L504

Otherwise, roughnums will cause issues.